.gitignore

```diff
@@ -2,7 +2,6 @@ python
 node_modules
 secrets.js
 package-lock.json
```

Comment:
I think `package-lock.json` should be under version control

here is why:
https://docs.npmjs.com/files/package-lock.json